Improve docs for first time developer setup (#623)
flowforge/flowforge: README.md

# Build the front end
npm run build

# Start the application
npm start
```

If you're actively developing for the FlowForge core, then use `npm run serve` to run the application
with automatic rebuilds of the front-end, and restarts of the application when
it is modified.

> If you intend on developing for the additional repositories, please refer to [the docs](https://github.com/flowforge/flowforge/tree/dev-docs#readme) 

Reviewer: ```suggestion
If you intend on developing for the additional repositories, please refer to [the docs](https://github.com/flowforge/flowforge/tree/dev-docs#readme) 
```
Where should this be linked to? Currently it points at the same file but on your feature branch (that will get deleted once this PR is merged).

Author: ```suggestion
If you intend on developing for the additional repositories, please refer to [the docs](https://flowforge.com/docs/contribute/) 
```

Well spotted. Should be `https://flowforge.com/docs/contribute/`

Author: Or should it be `https://github.com/flowforge/flowforge/tree/main/docs/contribute#readme`? 🤔

> for detailed developer setup instructions

Reviewer 2: ```suggestion
for detailed developer setup instructions.
```
